db/Dockerfile

```diff
@@ -1,25 +1,25 @@
-FROM postgres:9.4
+FROM postgres:9.4-alpine
+
+ENV DEFAULT_TZ UTC
 
 # Install some packages to use WAL
-RUN apt-get update \
-    && apt-get install -y \
-      build-essential \
+RUN apk add --no-cache --virtual .build-deps \
       curl \
```

Comment:
I know I sound maniac, but can you order package list in alphabetical order please (like it was before) ? It is a good practice :)

Reply:
fix